radish34/api/src/install/checkWallet.js

```diff
@@ -3,10 +3,9 @@ import { getWallet } from '../utils/wallet';
 
 export default async () => {
   const wallet = await getWallet();
-
   if (wallet) {
     console.log(`Loading wallet with address ${wallet.signingKey.address}`);
-    setOrganizationWalletAddress(wallet.signingKey.address);
+    setOrganizationWalletAddress(address);
```

Comment:
I don't think this will work. What does `address` resolve to?

Reply:
Instead of `address` it should have been `wallet.signingKey.address`, fixed that.